### DISCUSSION TITLE
Include Java bindings in release

### DISCUSSION THREAD

**Discussion by Selebrator** [cvc5/cvc5#9916] [Ideas]
Hi, I'm [depending](https://github.com/Selebrator/ColorUnfolder) on the Java Bindings for cvc5.
I want to automate my dependency resolution and set up automated builds/tests (CI).
As far as I can tell, there is no distribution of cvc5 with Java Bindings; did I miss something?
I would appreciate the distribution found at the Github releases to include the Java bindings, because then I and others, e.g. [package maintainers](https://aur.archlinux.org/packages/cvc5-bin), would be able to easily obtain them. Building from source takes roughly 10 minutes on 8 cores for me. I don't care about the Python bindings, but they would be nice to have as well I assume.

Greetings from the University of Oldenburg, Germany

**Comment by Selebrator** [comment]
I got the GitHub CI running ([see](https://github.com/Selebrator/ColorUnfolder/tree/d3aacfc83469c55683c87d1c786e151fe32cd0ea) `/lib`), to demonstrate that all I need is `cvc5.jar`, and `libcvc5jni.so` and its dependencies, which are already in the release.

**Comment by mpreiner** [comment]
Hi @Selebrator! Thanks for your suggestion. We'll include all bindings in future releases.

**Comment by Selebrator** [comment]
That's great news. Thank you!

**Reply by daniel-larraz** [reply]
Hi @Selebrator . This is to let you know that the latest release (1.1.1) includes Java Bindings for Linux and macOS in the "shared" packages.
